packages/sources/amberdata/src/index.ts

```diff
@@ -1,5 +1,5 @@
 import { expose } from '@chainlink/ea-bootstrap'
 import { makeExecute } from './adapter'
-import { makeConfig, NAME } from './config'
+import { makeConfig, makeWSHandler, NAME } from './config'
 
-export = { NAME, makeExecute, makeConfig, ...expose(makeExecute()) }
+export = { NAME, makeExecute, makeConfig, ...expose(makeExecute(), makeWSHandler(makeConfig())) }
```

Comment:
Token Allocations is importing this package, which on import runs `makeWSHandler(makeConfig())`, which throws an error in `makeConfig`because `API_KEY` isn't provided.

Reply:
Aha I see, thanks!